pip/pip-275.md

```diff
@@ -0,0 +1,59 @@
+<!--
+RULES
+* Never place a link to an external site like Google Doc. The proposal should be in this issue entirely.
+* Use a spelling and grammar checker tools if available for you (there are plenty of free ones).
+
+PROPOSAL HEALTH CHECK
+I can read the design document and understand the problem statement and what you plan to change *without* resorting to a couple of hours of code reading just to start having a high level understanding of the change.
+
+THIS COMMENTS
+Please remove them when done.
+-->
+
+# Background knowledge
```

Comment:
I can't even start to read this PIP as vital background information needs to be included.
Can you please read the comment for the background knowledge section and add it accordingly?

Reply:
OK. PTAL